scss/components/_side-nav.scss

```diff
@@ -30,6 +30,15 @@
   &:last-child {
     padding-bottom: 0;
   }
+
+  &.is-disabled {
```

Comment:
Small thing: You shouldn't need this if you just don't wrap the inner element in a `.side-nav__link` class. You could just leave it `<l class="is-disabled side-nav__item"i>TBD</li>` (which is what we do elsewhere.

I don't mind having this in there if that's the way you'd rather go though.